app.py

```
@@ -8,21 +8,15 @@
 import time
 import signal
 import sys
-import shutil
+
 
 app = Flask(__name__)
 app.secret_key = 'your_secret_key'
```

Ruff (0.7.0)

Possible hardcoded password assigned to: "secret_key"

(S105)